docs/usage/autoscaling/shoot_pod_autoscaling_best_practices.md

```diff
@@ -157,7 +157,7 @@ spec:
     kind:       Deployment
     name:       foo-deployment
   updatePolicy:
-    updateMode: "Auto"
+    updateMode: "Recreate"
```

Comment:
We could also adapt https://github.com/gardener/gardener/blob/9ed35fb18ca49bbffac239d356e8b712dc4ee763/docs/usage/autoscaling/shoot_pod_autoscaling_best_practices.md#L175-L178 to describe the `Recreate` mode and make it clear that the `Auto` mode is deprecated.

Reply:
Thanks for pointing to the guide. The latest updates include:

- A _deprecation_ note to the `Auto` _update mode_ section as well as an update to its description from the [upstream quickstart guide](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/docs/quickstart.md)
- A new `Recreate` section as it was missing from the guide, but it is now the _default_ update mode with VPA [v1.5.0](https://github.com/kubernetes/autoscaler/releases/tag/vertical-pod-autoscaler-1.5.0)